packages/app-store/_components/OmniInstallAppButton.tsx

```diff
@@ -61,7 +61,6 @@ export default function OmniInstallAppButton({
                 type: app.type,
                 variant: app.variant,
                 slug: app.slug,
-                isOmniInstall: true,
```

Comment:
Why is it removed? 

I see usage of `isOmniInstall` still in the mutation code https://github.com/calcom/cal.com/blob/45fe2a727eaf774b6e972bbc432602faae50fba6/packages/app-store/_utils/useAddAppMutation.ts#L57

Reply:
Heads up that https://github.com/calcom/cal.com/pull/14450 was merged early into this branch so a lot of context was lost. 

I thought this prop wasn't being used but good catch.